source/includes/search/_index.md

```diff
@@ -477,4 +493,37 @@ Our **Keyword Search** function `(search=searchterm)` allows you to search text
 + <code>shortDescription</code>
 + <code>longDescription</code>
 + <code>features.feature</code>
-+ <code>details.value</code>
\ No newline at end of file
++ <code>details.value</code>
+
+## Search on Reviews
+```shell
+curl "https://api.bestbuy.com/v1/products(customerReviewAverage>=4&customerReviewCount>100)?show=customerReviewAverage,customerReviewCount,name,sku&format=json&apiKey=YourAPIKey"
+```
+```javascript
+var bby = require('bestbuy')('YourAPIKey');
+bby.products('customerReviewAverage>=4&customerReviewCount>100',{show:'customerReviewAverage,customerReviewCount,name,sku'}).then(function(data){
+  console.log(data);
+});
+```
+```json-doc
+{
+ "products": [
+    {
+      "customerReviewAverage": "4.1",
+      "customerReviewCount": 411,
+      "name": "Insignia™ - Soundbar Home Theater Speaker System",
+      "sku": 4841342
+    },
+    {
+      "customerReviewAverage": "4.3",
+      "customerReviewCount": 411,
+      "name": "Sunpak - PlatinumPlus 6000PG 61\" Tripod",
+      "sku": 1205204
+    }
+  ]
+}
+```
+
+To search based on review criteria you can use the `customerReviewAverage` and/or the `customerReviewCount` attributes. You can also limit the product information returned using our `show` functionality. *HINT: You can specify additional attributes in your search or to be included in the response document for most attributes in the Products API.*
```

Comment:
How about rewriting the first sentence something like this, "To search for products based on customer review criteria, you can use...."